Frends.SFTP.ListFiles/Frends.SFTP.ListFiles/Frends.SFTP.ListFiles.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.ComponentModel;
+using Renci.SshNet;
+using System.Net.Sockets;
+using Renci.SshNet.Common;
+using System.Text.RegularExpressions;
+using Frends.SFTP.ListFiles.Definitions;
+
+#pragma warning disable 1591
+
+namespace Frends.SFTP.ListFiles
+{
+    public class SFTP
+    {
+        /// <summary>
+        /// Reads a file with SFTP connection.
+        /// Documentation: https://github.com/FrendsPlatform/Frends.SFTP.ReadFile
```

Comment:
Please modify this to `[Documentation](link)` format, since then the link will be attached to the Documentation-text and not as separate link, like this: [Documentation](https://github.com/FrendsPlatform/Frends.SFTP.ReadFile). Also, the link should point to https://tasks.frends.com/tasks#frends-tasks/Frends.SFTP.ListFiles according to latest guidelines.

Reply:
Fixed